ui/components/DatabaseSummary.js

```diff
@@ -121,62 +102,30 @@ const DatabaseSummary = (props) => {
   };
 
   const columns = [
-    {
-      name: 'name',
-      label: 'Name',
-    },
-    {
-      name: 'count',
-      label: 'Count',
-    },
+    { name: 'name', label: 'Name' },
+    { name: 'count', label: 'Count' },
   ];
 
-  const [tableCols, updateCols] = useState(columns);
-
-  const [columnVisibility] = useState(() => {
-    // Initialize column visibility based on the original columns' visibility
-    const initialVisibility = {};
-    columns.forEach((col) => {
-      initialVisibility[col.name] = col.options?.display !== false;
-    });
-    return initialVisibility;
-  });
-
-  const customInlineStyle = {
-    marginBottom: '0.5rem',
-    marginTop: '1rem',
-  };
-
   return (
-    <>
-      <div className={StyleClass.toolWrapper} style={customInlineStyle}>
```

Comment:
its good that you have removed the class uses but have you ensure that are converted to style components and there styles are not lost?